app/models/authorization.rb

```diff
@@ -0,0 +1,35 @@
+# encoding: utf-8
+
+class Authorization
+  attr_accessor :token
+  attr_reader :token_auth_accounts
+
+  def initialize(token)
+    @token = token
+  end
+
+  def id
```

Comment:
I added this because the AuthorizationRepresenter wants an id to build links ... misguided?